lib/testing/src/lib/protractor/core/pages/form/widgets/tab.page.ts

```diff
@@ -35,4 +35,8 @@ export class TabPage {
     async checkTabIsNotDisplayedByLabel(tabLabel): Promise<void> {
         await BrowserVisibility.waitUntilElementIsNotVisible(element(by.cssContainingText('.mat-tab-label-content', tabLabel)));
     }
+
+    async checkWarningIconIsDisplayed(): Promise<void> {
+        await BrowserVisibility.waitUntilElementIsVisible(element(by.css('[data-automation-id="adf-content-node-selector-disabled-tab-info-icon"]')));
```

Comment:
This thing you can do as TestElement and it will be a one line method then, so:
```
public disabledContentNodeSelectorTabInfoIcon = TestElement.byCss('[data-automation-id="adf-content-node-selector-disabled-tab-info-icon"]');
```

and then in spec file:
```
await <pageObject>.disabledContentNodeSelectorTabInfoIcon.waitVisible()
```